src/components/Layout/Layout.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Layout from '.';
+
+describe('Layout', () => {
+  const children = <h1>hello</h1>;
+
+  beforeEach(() => {
+    render(<Layout>{children}</Layout>);
+  });
+
+  it('renders all children', () => {
+    const layout = screen.getByTestId('layout');
```

Comment:
Layout component renders a main html tag.  This is reachable by role : `main`. Thus the data-testid is not necessary.